### ISSUE TITLE
[Issue] ISNT using significant server time

### ISSUE THREAD

**Issue by Renari** [nanite/ItShallNotTick#7]
### Mod version

1.0.20-build.21

### Forge mod loader version

40.1.51

### Repeatable?

- [X] Yes

### Link to log files

Issue appears to be persistent.

https://spark.lucko.me/x4bioDjWjc

### Description of the issue you are having

ISNT taking a significant amount of tick time specifically in the tick zombie call.

### Information

- [X] I have provided as much information as possible

**Comment by Gaz492** [comment]
Not much I can do to fix this, while my mod shows up as using up a lot of the tick time it's only doing so due to how the mod works, in reality it's Minecraft that's the problem.
This can be demonstrated by disabling my mod then running the profiler again, you will see that the lag is caused by the guardedEntityTick method, you may even notice that it uses up more of the tick time.

TL;DR not actually my mod causing the large tick time usage, just Minecraft being bad with how it does mob AI
